go-app-ussd.js

```diff
@@ -1238,10 +1238,14 @@ go.app = function() {
                 error_question:
                     $("Sorry, we're not sure which suburb you mean. Please re-enter your location again carefully and make sure you use the correct spelling."),
                 next: 'state_report_submit_critical',
-                next_text: 'More',
-                previous_text: 'Back',
+                next_text: $('More'),
+                previous_text: $('Back'),
+                retry_text: $('Retry'),
+                skip_text: $('Skip'),
+                
 
                 map_provider: new OpenStreetMap({
+                    api_key: 'XXX',
```

Comment:
We need to create a ticket for fixing this properly (location searches won't work until it is fixed).